sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -1200,7 +1333,7 @@ case class UnBase64(child: Expression) extends UnaryExpression with ImplicitCast
  * If either argument is null, the result will also be null.
  */
 @ExpressionDescription(
-  usage = "_FUNC_(bin, str) - Decode the first argument using the second argument character set.")
+  usage = "_FUNC_(bin, str) - Decodes the first argument using the second argument character set.")
```

Comment:
`_FUNC_(bin, str) - Decodes the first argument using the second argument character set.`
->
```
_FUNC_(expr, charset) - Decodes the string `expr` using the character set `charset`.
```